src/components/atoms/Rating.js

```diff
@@ -0,0 +1,23 @@
+import { Flex, Text } from '@chakra-ui/react'
+import pt from 'prop-types'
+
+export function Rating({ value }) {
```

Comment:
We should add default props. They are moving towards deprecating the old way.
```
// Old way
Rating.defaultProps = {
value: 0
}
```
```suggestion
export function Rating({ value = 0 }) {
```